flowsa/methods/flowbysectormethods/CAP_HAP_national_2017.yaml

```diff
@@ -25,8 +25,8 @@ source_names:
           - "AREA HARVESTED"
         allocation_compartment: None
         allocation_from_scale: state
-        clean_allocation_fba: coa_irrigated_cropland_fba_cleanup
-        clean_allocation_fba_w_sec: disaggregate_coa_cropland_to_6_digit_naics
+        clean_allocation_fba: !script_function:EPA_NEI coa_irrigated_cropland_fba_cleanup
+        clean_allocation_fba_w_sec: !script_function:EPA_NEI disaggregate_coa_cropland_to_6_digit_naics
       activity_set_3: #Fuel Combustion - Industry #only allocating to 3digits
```

Comment:
for these clean functions, the script source is the allocation source, e.g.:
`!script_function:USDA_CoA_Cropland coa_irrigated_cropland_fba_cleanup`

Reply:
> @matthewlchambers I'll review the FBS per my comment above and push a few changes

Ah, of course. Thanks for taking care of that!